index.js

```diff
@@ -105,6 +98,30 @@ function loadAndValidateEnvironment (_opts) {
   return merge
 }
 
+function chooseAjvInstance (defaultInstance, ajvOpts) {
+  if (!ajvOpts) {
+    return defaultInstance
+  } else if (typeof ajvOpts === 'object' && typeof ajvOpts.customOptions === 'function') {
+    const ajv = ajvOpts.customOptions(getDefaultInstance())
```

Comment:
Is there a reason why not doing
```js
const ajv = ajvOpts.customOptions(defaultInstance)
```

Reply:
Yes, it is: the `defaultInstance` is a global variable one and giving the user to change it with the customOptions function will change the default for everyone.

Doing so would break the `fastify-env` plugin, but creating a new default instance before customizing it would change only the new variable.

I did not remove the global instance, instead, because it would be a downgrade for users that already uses this module and don't need this new option